test/Common/Microsoft.Test.OData.DependencyInjection/TestContainerBuilder.cs

```diff
@@ -12,37 +12,9 @@
 
 namespace Microsoft.Test.OData.DependencyInjection
 {
-    public class TestContainerBuilder : IContainerBuilder
+    public class TestContainerBuilder 
     {
-        private readonly IServiceCollection services = new ServiceCollection();
-
-        public IContainerBuilder AddService(
-            Microsoft.OData.ServiceLifetime lifetime,
-            Type serviceType,
-            Type implementationType)
-        {
-            Debug.Assert(serviceType != null, "serviceType != null");
-            Debug.Assert(implementationType != null, "implementationType != null");
-
-            services.Add(new ServiceDescriptor(
-                serviceType, implementationType, TranslateServiceLifetime(lifetime)));
-
-            return this;
-        }
-
-        public IContainerBuilder AddService(
-            Microsoft.OData.ServiceLifetime lifetime,
-            Type serviceType,
-            Func<IServiceProvider, object> implementationFactory)
-        {
-            Debug.Assert(serviceType != null, "serviceType != null");
-            Debug.Assert(implementationFactory != null, "implementationFactory != null");
-
-            services.Add(new ServiceDescriptor(
-                serviceType, implementationFactory, TranslateServiceLifetime(lifetime)));
-
-            return this;
-        }
+        public IServiceCollection services = new ServiceCollection();

```

Comment:
```suggestion
        public IServiceCollection Services { get; } = new ServiceCollection();
```

Reply:
`services` was a private readonly field. To expose the same publicly, the right thing is to make it a property and use Pascal case